components/ModalSketch.js

```diff
@@ -0,0 +1,128 @@
+import styled from "styled-components";
+import Modal from "./Modal";
+import Image from "next/image";
+import UploadIcon from "@/public/upload.svg";
+import BinIcon from "@/public/binIcon.svg";
+import { useAtom } from "jotai";
+import { statusUploadAtom, showEditImageAtom } from "@/lib/atom";
+
+export default function ModalDelete({
+  image,
+  showModalSketch,
+  handleClose,
+  handleImageChange,
+  handleDeleteImage,
+}) {
+  const [editImage, setEditImage] = useAtom(showEditImageAtom);
+  const [statusUpload, setStatusUpload] = useAtom(statusUploadAtom);
+
+  return (
+    <Modal show={showModalSketch}>
```

Comment:
modal seems to be an interesting package :) is it for the pop ups?

Reply:
yes i like to work with them